jukebox/src/main/webapp/js/youtubePlayerScript.js

```diff
@@ -7,19 +7,26 @@
 
     var player;
     var display_container = document.getElementById("display_container");
+    var isVideoPlaying = true; //Keeps track of whether video is playing; ignores user pauses
 
     //Finds the youtube video and creates an iframe
     function getYouTube(vidID, nextSong) {
         //Case where there are no videos
         if(vidID == NO_VIDEO){
             display_container.innerHTML = `No songs in queue.`;
+            //If player has been created, stop the video
+            if (player) {
+                stopVideo();
+            }
+            isVideoPlaying = false;
```

Comment:
isVideoPlaying looks to be set to whenever the vidId is unset - do we need this variable?

Reply:
Yeah, I see how that variable was a little bit redundant. I've now made a global vidID variable instead and check if the video is playing with that now.